### ISSUE TITLE
incompatability

### ISSUE THREAD

**Issue by nuclearreactormadefromcheese** [SelfControlApp/selfcontrol#888]
its currently not compatable with opera GX could you pretty please witha cherry on top you make this compatible with opra GX

**Comment by valimikayilov** [comment]
I don't think it is the selfcontroll app problem. The issue is opera GX is ignoring hosts file, so I would advise to just change the browser.
